site/content/docs/5.2/content/tables.md

```diff
@@ -643,6 +643,10 @@ The selection behavior isn't implemented yet. This feature will be delivered wit
 
 <div>
   <table class="table table-sm table-hover table-responsive">
+    <colgroup>
```

Comment:
Not a big fan of adding a `colgroup` here. Can't be handle everything by using the same thing that was done in v4?

Using a `.has-checkbox` in `<table>` plus a rule similar (+ some adjustements) to:

```scss
// Boosted mod
.has-checkbox tr > :first-child {
  width: $spacer;
}
// end mod
```